sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -976,7 +976,7 @@ class HiveDDLSuite
           exception = intercept[AnalysisException] {
             sql(s"ALTER TABLE $oldViewName RECOVER PARTITIONS")
           },
-          errorClass = "UNSUPPORTED_VIEW_OPERATION.WITH_SUGGESTION",
```

Comment:
The old logic will prompt `Please use ALTER VIEW instead.` here, 
Because the syntax of `ALTER VIEW ... RECOVER PARTITIONS` is not supported on view, prompts can actually cause misunderstandings.
which is incorrect. 
I have checked the rationality of the error prompts returned by all test cases.

Reply:
Other modifications are similar.